langchain4j/src/main/java/dev/langchain4j/service/AiServices.java

```diff
@@ -171,6 +175,16 @@ public static <T> AiServices<T> builder(Class<T> aiService) {
         return new DefaultAiServices<>(context);
     }
 
+    public AiServices<T> systemMessages(Function<Object, String> systemMessagesProvider) {
```

Comment:
```suggestion
    public AiServices<T> systemMessageProvider(Function<Object, String> systemMessageProvider) {
```
To keep the same style with e.g. `chatMemoryProvider`